docs/source/tutorials/muon_GUI_course/the_tabs_home.rst

```diff
@@ -0,0 +1,70 @@
+.. _the_tabs_home:
+
+===============
+The Tabs - Home
+===============
+
+.. index:: The Tabs - Home
+
+.. contents:: Table of Contents
+  :local:
+
+Home
+====
+
+When launched, the Muon Analysis GUI defaults to the Home tab. This tab allows: 
+
+* data files to be loaded
+* run information to be viewed
+* detector groupings to be selected
```

Comment:
I think we should replace this with something about deadtime, rebin, first good etc.